src/DIPS.Xamarin.UI/Controls/RadioButtonGroup/RadioButtonGroup.xaml.cs

```diff
@@ -0,0 +1,323 @@
+        {
+            InitializeComponent();
+        }
+
+        /// <summary>
+        ///     The color for each radio button when it is not selected
+        ///     This is a bindable property
+        /// </summary>
+        public Color DeSelectedColor
+        {
+            get => (Color)GetValue(DeSelectedColorProperty);
+            set => SetValue(DeSelectedColorProperty, value);
+        }
+
+        /// <summary>
+        ///     The member path to use for the label for each radio button. This have to point at a property in object of
+        ///     <see cref="ItemsSource" />
+        ///     This is a bindable property
+        /// </summary>
+        /// <remarks>If this is not set, the ToString() of each item in the <see cref="ItemsSource" /> will be used</remarks>
+        public string DisplayMemberPath
+        {
+            get => (string)GetValue(DisplayMemberPathProperty);
+            set => SetValue(DisplayMemberPathProperty, value);
+        }
+
+        /// <summary>
+        ///     The collection of items that should be used for each radio button, this should have a property that have to be
+        ///     pointed at by the <see cref="DisplayMemberPath" />
+        ///     This is a bindable property
+        /// </summary>
+        public IList ItemsSource
+        {
+            get => (IList)GetValue(ItemsSourceProperty);
+            set => SetValue(ItemsSourceProperty, value);
+        }
+
+        /// <summary>
+        ///     The color for each radio button when it is selected
+        ///     This is a bindable property
+        /// </summary>
+        public Color SelectedColor
+        {
+            get => (Color)GetValue(SelectedColorProperty);
+            set => SetValue(SelectedColorProperty, value);
+        }
+
+        /// <summary>
+        ///     The selected radio button from the group. This value will be set when the user clicks a radio button and can be
+        ///     used to set it initially.
+        ///     This is a bindable property
+        /// </summary>
+        public object SelectedItem
+        {
+            get => GetValue(SelectedItemProperty);
+            set => SetValue(SelectedItemProperty, value);
+        }
+
+        /// <summary>
+        ///     The color of the separator that separates each radio button
+        ///     This is a bindable property
+        /// </summary>
+        public Color SeparatorColor
+        {
+            get => (Color)GetValue(SeparatorColorProperty);
+            set => SetValue(SeparatorColorProperty, value);
+        }
+
+        void IHandleRadioButtons.OnRadioButtonTapped(RadioButton tappedRadioButton)
+        {
+            if (SelectedItem == tappedRadioButton.Identifier) return;
+
+            var selectedObject = ItemsSource.Cast<object>().FirstOrDefault(item => tappedRadioButton.Identifier == item);
+
+            if (selectedObject == null) return;
+
+            SelectedItem = selectedObject;

```

Comment:
`Boxview` has no border posibilities. This will lead to adding three boxes to create the border effect. An issue with Background transparancy will occur with this solution.

`Frame` cornerradius is faulty on iOS. (https://github.com/xamarin/Xamarin.Forms/issues/5199) This will be a problem for us if we open up for the consumers to change size of our buttons.

This is why I ended up with `Button`. This will save us time the day we open up for size rezising.